src/App.js

```diff
@@ -1,43 +1,47 @@
+import React from "react";
+import { Route, Routes } from "react-router-dom";
+import Home from "./pages/Home";
+import WatchLater from "./pages/WatchLater";
+import History from "./pages/History";
+import MyPlaylists from "./pages/MyPlaylists";
+import SignUp from "./pages/SignUp";
+import LogIn from "./pages/LogIn";
+import Mockman from "mockman-js";
+import NotFound from "./pages/NotFound";
+import { RequireAuth } from "./pages/RequireAuth";
```

Comment:
You can create a seprate index file and exports all pages from there it is easy import all files in single line.

Reply:
Done in Upcoming PR